config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,38 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.2 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
```

Comment:
@aivils once rails upgrade is rolled out, will we go through these and enable the new defaults? Do you see any potential issues?

Reply:
@bladealslayer 
Yes, this is forgotten. I will check it out.